[2022-02-03 20:53:13,680] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2019-04-01T00:00:00+00:00 [queued]>
[2022-02-03 20:53:13,758] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2019-04-01T00:00:00+00:00 [queued]>
[2022-02-03 20:53:13,763] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 20:53:13,767] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-03 20:53:13,769] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 20:53:13,923] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-04-01 00:00:00+00:00
[2022-02-03 20:53:13,955] {standard_task_runner.py:52} INFO - Started process 6724 to run task
[2022-02-03 20:53:13,991] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag_5', 'download_dataset_task', 'scheduled__2019-04-01T00:00:00+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag_2.py', '--cfg-path', '/tmp/tmpqlsmcbzs', '--error-file', '/tmp/tmpm6wf2hy8']
[2022-02-03 20:53:13,995] {standard_task_runner.py:77} INFO - Job 38: Subtask download_dataset_task
[2022-02-03 20:53:14,670] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2019-04-01T00:00:00+00:00 [running]> on host 985445ce9491
[2022-02-03 20:53:15,136] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag_5
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-04-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-01T00:00:00+00:00
[2022-02-03 20:53:15,143] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-03 20:53:15,149] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'pwd']
[2022-02-03 20:53:15,231] {subprocess.py:85} INFO - Output:
[2022-02-03 20:53:15,333] {subprocess.py:89} INFO - /tmp/***tmpw_j3pzd4
[2022-02-03 20:53:15,364] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-03 20:53:17,281] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag_5, task_id=download_dataset_task, execution_date=20190401T000000, start_date=20220203T205313, end_date=20220203T205317
[2022-02-03 20:53:17,623] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 20:53:18,165] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
